aries-site/src/examples/templates/forms/managing-child-objects/CreateCluster.js

```diff
@@ -0,0 +1,238 @@
+    console.log(event.value);
+  };
+
+  return (
+    <Box alignSelf="center" gap="medium" width="medium">
+      <Heading level={2} margin="none">
+        Create Cluster
+      </Heading>
+      <Form value={formValues} onChange={onChange} onSubmit={onSubmit}>
+        <Box gap="medium">
+          <>
+            <FormField
+              htmlFor="cluster-name"
+              name="cluster-name"
+              label="Cluster name"
+              required
+              aria-required="true"
+            >
+              <TextInput id="cluster-name" name="cluster-name" />
+            </FormField>
+            <FormField
+              htmlFor="resource-manager"
+              name="resource-manager"
+              label="Distributed resoure manager"
+            >
+              <CheckBox
+                id="resource-manager"
+                name="resource-manager"
+                label="Use manager"
+              />
+            </FormField>
+            {formValues['resource-manager'] && (
+              <>
+                <FormField
+                  htmlFor="automation-level"
+                  name="automation-level"
+                  label="Automation level"
+                >
+                  <Select
+                    id="automation-level"
+                    name="automation-level"
+                    options={['Smart (recommended)', 'High', 'Medium', 'Low']}
+                    placeholder="Select ..."
+                  />
+                </FormField>
+                <FormField
+                  htmlFor="migration-threshold"
+                  name="migration-threshold"
+                  label="Migration threshold"
+                >
+                  <Box
+                    direction="row"
+                    gap="small"
+                    pad={{ horizontal: 'small', vertical: 'xsmall' }}
+                  >
+                    <Text>0%</Text>
+                    <RangeInput
+                      id="migration-threshold"
+                      name="migration-threshold"
+                      min={0}
+                      max={50}
+                    />
+                    <Text>50%</Text>
+                  </Box>
+                </FormField>
+              </>
+            )}
+          </>
+          <Box gap="small">
+            <Heading level={3} margin="none">
+              Hosts
+            </Heading>
+            <FormChildObjects
+              collection={{
+                name: 'Hosts',
+                itemName: 'host',
+                parentName: 'cluster',
+              }}
+              fields={INPUT_MAP}
+              level={4}
+              onAdd={handleAdd}
+              onRemove={handleRemove}
+              onRemoveAll={handleRemoveAll}
+              primaryKey="host"
+              required
+              summarize={['cpu', 'memory']}
+              values={formValues.hosts}
+            />
+          </Box>
+          <Box direction="row" gap="xsmall">
```

Comment:
In other examples recently I've been doing gap="small" -- wondering which we want to standardize on

Reply:
Yes, we should standardize. This would be a good place to have `<ButtonGroup />` which among other things (e.g. responsive behavior) would standardize the gap via theme.

I'm going to create a local `<ButtonGroup />` template to start using as a prototype and start standardizing.